backend/tests/Designer.Tests/Controllers/AppDevelopmentController/UpdateLayoutSetTests.cs

```diff
@@ -45,10 +45,9 @@ public class UpdateLayoutSetTests(WebApplicationFactory<Program> factory)
             LayoutSets layoutSetsAfter = await GetLayoutSetsFile(org, targetRepository, developer);
 
             layoutSetsBefore.Schema.Should().NotBeNull();
-            layoutSetsBefore.Sets.Should().HaveCount(3);
             Assert.False(layoutSetsBefore.Sets.Exists(set => set.DataType == newLayoutSetConfig.DataType));
             layoutSetsAfter.Schema.Should().NotBeNull();
-            layoutSetsAfter.Sets.Should().HaveCount(3);
+            layoutSetsBefore.Sets.Count.Should().Be(layoutSetsAfter.Sets.Count);
```

Comment:
Can this be written like this? 🤔 
```suggestion
            layoutSetsBefore.Sets.Should().HaveCount(layoutSetsAfter.Sets.Count);
```